packages/docs/src/common/propsStore.ts

```diff
@@ -12,10 +12,12 @@ export interface IInheritedPropertyEntry extends ITsPropertyEntry {
 }
 
 export class PropsStore {
-    constructor(private props: { [name: string]: ITsInterfaceEntry }) {}
+    constructor(private props: { [name: string]: ITsInterfaceEntry }) { }
 
     public getInterface = (name: string) => {
-        return this.props[name];
+        // remove generics from end of name
+        const actualName = /^(\w+)<?/.exec(name)[1];
```

Comment:
hmm this feels like it should probably be fixed at the ts-quick-docs layer? interfaces have __names__ and __type params__, which we could preserve in a data object and avoid this regex